feat: For npm publish

Brew-Brew opened the pull request:
---
name: ☄️ For npm publish
about: Pull Request! 
---

## 🤖Summary
- edit webpack config for publish
- publish test
- edit npmignore

**packages**
https://www.npmjs.com/search?q=%40emplodies
![스크린샷 2019-07-28 오후 12 51 24](https://user-images.githubusercontent.com/26598542/62001986-72918300-b136-11e9-8dd9-d311343eaf25.png)

## 📝  Post actions
- ~deal with various javascript environment~
- npmignore bug to reduce package sizes
- deal with each function's import error case

## examples
- test normalizekeyword
![스크린샷 2019-07-28 오전 11 57 48](https://user-images.githubusercontent.com/26598542/62001620-60f8ad00-b12f-11e9-98c3-e415b5f21c80.png)

Brew-Brew committed:
feat: :wrench: add publish script

Brew-Brew committed:
Publish

 - @emplody/spaceship.asyncState@0.0.1
 - @emplody/spaceship.detectBrowser@0.0.1
 - @emplody/spaceship.normalizeKeyword@0.0.1
 - @emplody/spaceship.objToParams@0.0.1
 - @emplody/spaceship.paramsToObj@0.0.1
 - @emplody/spaceship.scriptLoader@0.0.1
 - @emplody/spaceship.utilExample@0.0.1
 - @emplody/spaceship@0.0.1

Brew-Brew committed:
feat: :wrench: add publish config public to packages

Brew-Brew committed:
Publish

 - @emplody/spaceship.asyncState@0.0.2
 - @emplody/spaceship.detectBrowser@0.0.2
 - @emplody/spaceship.normalizeKeyword@0.0.2
 - @emplody/spaceship.objToParams@0.0.2
 - @emplody/spaceship.paramsToObj@0.0.2
 - @emplody/spaceship.scriptLoader@0.0.2
 - @emplody/spaceship.utilExample@0.0.2
 - @emplody/spaceship@0.0.2

Brew-Brew committed:
feat: :wrench: change org name

Brew-Brew committed:
feat: :wrench: edit version of utils to initialize

Brew-Brew committed:
Publish

 - @emplodies/spaceship.asyncState@0.0.1
 - @emplodies/spaceship.detectBrowser@0.0.1
 - @emplodies/spaceship.normalizeKeyword@0.0.1
 - @emplodies/spaceship.objToParams@0.0.1
 - @emplodies/spaceship.paramsToObj@0.0.1
 - @emplodies/spaceship.scriptLoader@0.0.1
 - @emplodies/spaceship.utilExample@0.0.1
 - @emplodies/spaceship@0.0.1

Brew-Brew committed:
feat: :wrench: edit name of util for npm naming rules not allowed uppercase

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.2
 - @emplodies/spaceship.detect-browser@0.0.2
 - @emplodies/spaceship.normalize-keyword@0.0.2
 - @emplodies/spaceship.obj-to-params@0.0.2
 - @emplodies/spaceship.params-to-obj@0.0.2
 - @emplodies/spaceship.script-loader@0.0.2
 - @emplodies/spaceship.util-example@0.0.2
 - @emplodies/spaceship@0.0.2

Brew-Brew committed:
feat: :wrench: add prepublish build and edit npmignore to reduce package size

Brew-Brew committed:
feat: :wrench: editspaceship index.js

Brew-Brew committed:
Publish

 - @emplodies/spaceship@0.0.3

Brew-Brew committed:
feat: :wrench: edit npmignore

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.3
 - @emplodies/spaceship.detect-browser@0.0.3
 - @emplodies/spaceship.normalize-keyword@0.0.3
 - @emplodies/spaceship.obj-to-params@0.0.3
 - @emplodies/spaceship.params-to-obj@0.0.3
 - @emplodies/spaceship.script-loader@0.0.3
 - @emplodies/spaceship.util-example@0.0.3
 - @emplodies/spaceship@0.0.4

Brew-Brew committed:
feat: :wrench: edit npmignore

Brew-Brew committed:
feat: :wrench: edit webpack config

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.4
 - @emplodies/spaceship.detect-browser@0.0.4
 - @emplodies/spaceship.normalize-keyword@0.0.4
 - @emplodies/spaceship.obj-to-params@0.0.4
 - @emplodies/spaceship.params-to-obj@0.0.4
 - @emplodies/spaceship.script-loader@0.0.4
 - @emplodies/spaceship.util-example@0.0.4
 - @emplodies/spaceship@0.0.5

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.5
 - @emplodies/spaceship.detect-browser@0.0.5
 - @emplodies/spaceship.normalize-keyword@0.0.5
 - @emplodies/spaceship.obj-to-params@0.0.5
 - @emplodies/spaceship.params-to-obj@0.0.5
 - @emplodies/spaceship.script-loader@0.0.5
 - @emplodies/spaceship.util-example@0.0.5
 - @emplodies/spaceship@0.0.6

Brew-Brew committed:
feat: :wrench: edit webpack config for library

SoYoung210 commented on .npmignore:
```diff
@@ -1,2 +1,5 @@
+**/package.json
+**/index.js
+**/test/
+!**/dist
```
i'll add `eslint eol-last: ["error", "always"]` option in eslint ㅎㅎ

Brew-Brew replied:
🙏 👀

Brew-Brew committed:
feat: :bug: edit repo name

Brew-Brew committed:
feat: :art: merge conflict

Brew-Brew committed:
feat: :sparkles: deal with various js import environment

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.6
 - @emplodies/spaceship.detect-browser@0.0.6
 - @emplody/spaceship.getPadNumber@0.0.1
 - @emplody/spaceship.isMobile@0.0.1
 - @emplodies/spaceship.normalize-keyword@0.0.6
 - @emplodies/spaceship.obj-to-params@0.0.6
 - @emplodies/spaceship.params-to-obj@0.0.6
 - @emplodies/spaceship.script-loader@0.0.6
 - @emplody/spaceship.toTimeString@0.0.1
 - @emplody/spaceship.uniqueKey@0.0.1
 - @emplodies/spaceship.util-example@0.0.6
 - @emplodies/spaceship@0.0.7

Brew-Brew committed:
feat: :bug: edit org npm name

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.7
 - @emplodies/spaceship.detect-browser@0.0.7
 - @emplodies/spaceship.getPadNumber@0.0.2
 - @emplodies/spaceship.isMobile@0.0.2
 - @emplodies/spaceship.normalize-keyword@0.0.7
 - @emplodies/spaceship.obj-to-params@0.0.7
 - @emplodies/spaceship.params-to-obj@0.0.7
 - @emplodies/spaceship.script-loader@0.0.7
 - @emplodies/spaceship.toTimeString@0.0.2
 - @emplodies/spaceship.uniqueKey@0.0.2
 - @emplodies/spaceship.util-example@0.0.7
 - @emplodies/spaceship@0.0.8

Brew-Brew committed:
feat: :bug: edit org name and add publish config

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.8
 - @emplodies/spaceship.detect-browser@0.0.8
 - @emplodies/spaceship.get-pad-number@0.0.3
 - @emplodies/spaceship.is-mobile@0.0.3
 - @emplodies/spaceship.normalize-keyword@0.0.8
 - @emplodies/spaceship.obj-to-params@0.0.8
 - @emplodies/spaceship.params-to-obj@0.0.8
 - @emplodies/spaceship.script-loader@0.0.8
 - @emplodies/spaceship.to-time-string@0.0.3
 - @emplodies/spaceship.unique-key@0.0.3
 - @emplodies/spaceship.util-example@0.0.8
 - @emplodies/spaceship@0.0.9

Brew-Brew committed:
feat: :sparkles: add default true to plugin

Brew-Brew committed:
feat: :sparkles: add default true to plugin

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.9
 - @emplodies/spaceship.detect-browser@0.0.9
 - @emplodies/spaceship.get-pad-number@0.0.4
 - @emplodies/spaceship.is-mobile@0.0.4
 - @emplodies/spaceship.normalize-keyword@0.0.9
 - @emplodies/spaceship.obj-to-params@0.0.9
 - @emplodies/spaceship.params-to-obj@0.0.9
 - @emplodies/spaceship.script-loader@0.0.9
 - @emplodies/spaceship.to-time-string@0.0.4
 - @emplodies/spaceship.unique-key@0.0.4
 - @emplodies/spaceship.util-example@0.0.9
 - @emplodies/spaceship@0.0.10

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.10
 - @emplodies/spaceship.detect-browser@0.0.10
 - @emplodies/spaceship.get-pad-number@0.0.5
 - @emplodies/spaceship.is-mobile@0.0.5
 - @emplodies/spaceship.normalize-keyword@0.0.10
 - @emplodies/spaceship.obj-to-params@0.0.10
 - @emplodies/spaceship.params-to-obj@0.0.10
 - @emplodies/spaceship.script-loader@0.0.10
 - @emplodies/spaceship.to-time-string@0.0.5
 - @emplodies/spaceship.unique-key@0.0.5
 - @emplodies/spaceship.util-example@0.0.10
 - @emplodies/spaceship@0.0.11

Brew-Brew committed:
feat: :sparkles: add npm ignore to each repo

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.11
 - @emplodies/spaceship.detect-browser@0.0.11
 - @emplodies/spaceship.get-pad-number@0.0.6
 - @emplodies/spaceship.is-mobile@0.0.6
 - @emplodies/spaceship.normalize-keyword@0.0.11
 - @emplodies/spaceship.obj-to-params@0.0.11
 - @emplodies/spaceship.params-to-obj@0.0.11
 - @emplodies/spaceship.script-loader@0.0.11
 - @emplodies/spaceship.to-time-string@0.0.6
 - @emplodies/spaceship.unique-key@0.0.6
 - @emplodies/spaceship.util-example@0.0.11
 - @emplodies/spaceship@0.0.12

Brew-Brew committed:
Publish

 - @emplodies/spaceship.async-state@0.0.12
 - @emplodies/spaceship.detect-browser@0.0.12
 - @emplodies/spaceship.get-pad-number@0.0.7
 - @emplodies/spaceship.is-mobile@0.0.7
 - @emplodies/spaceship.normalize-keyword@0.0.12
 - @emplodies/spaceship.obj-to-params@0.0.12
 - @emplodies/spaceship.params-to-obj@0.0.12
 - @emplodies/spaceship.script-loader@0.0.12
 - @emplodies/spaceship.to-time-string@0.0.7
 - @emplodies/spaceship.unique-key@0.0.7
 - @emplodies/spaceship.util-example@0.0.12
 - @emplodies/spaceship@0.0.13
